docs/02-_Palindrom.md

```diff
@@ -36,30 +36,42 @@ var palindrom = new Palindrom({attribute: value});
 Attribute              | Type          | Default                | Description
 ---                    | ---           | ---                    | ---
 `remoteUrl`            | **`String`**      |  **Required**          | PATCH server URL
-`onStateReset`         | *Function*    |                        | Called after initial state object is received from the server (NOT necessarily after WS connection was established), **it can be called again if the state was reset by a root-replacing patch**.
```

Comment:
Were the changes to the external API signed off? The first point of https://github.com/Palindrom/Palindrom/issues/224 says that you will be changing the internals of Palindrom, not the external API. It was not expected (at least by me) to change the external API in this PR and I think that such changes should be reverted.